hack/run-user-cluster-controller-manager.sh

```diff
@@ -72,6 +73,7 @@ set -x
   -kubeconfig=${KUBECONFIG_USERCLUSTER_CONTROLLER_FILE} \
   -metrics-listen-address=127.0.0.1:8087 \
   -health-listen-address=127.0.0.1:8088 \
+  -pprof-listen-address=":${PPROF_PORT}" \
   -namespace=${NAMESPACE} \
   -openvpn-server-port=${OPENVPN_SERVER_NODEPORT} \
   -cluster-url=${CLUSTER_URL} \
```

Comment:
some `shfmt -w hack/run-user-cluster-controller-manager.sh` will definitely benefit this file.

Reply:
I tried, with many different options, but none have actually improved the formatting in this file.